sst/nights/facilities_temperatures_reports.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>Maybe add a couple of sentences description here of what are the 3/4 temperature sensors about.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>